platform/viewer/cypress/integration/study-list/OHIFStudyList.spec.js

```diff
@@ -21,31 +21,42 @@ describe('OHIF Study List', function() {
       //cy.get('@modalities')
     });
 
-    it('searches Patient Name with exact string', function() {
+    it('Displays several studies initially', function () {
+      cy.waitStudyList();
+      cy.get('@searchResult2').should($list => {
+        expect($list.length).to.be.greaterThan(1);
+        expect($list).to.contain('Juno');
+        expect($list).to.contain('832040');
+      });
+    });
+
+    it('searches Patient Name with exact string', function () {
       cy.get('@PatientName').type('Juno');
       //Wait result list to be displayed
       cy.waitStudyList();
-      cy.get('@searchResult').should($list => {
+      cy.get('@searchResult2').should($list => {
```

Comment:
The correct value to read is searchResult2, which returns the correct list of tr items being found

Reply:
great